### PR TITLE
[MM-58377] Make sure we have an enterprise license when activating the plugin

### PR DESCRIPTION
**Pull request by JulienTant**
## Summary
Add license check when starting playbooks if the version is 2.0+

Also see the Mattermost PR: https://github.com/mattermost/mattermost/pull/27335

Note: there is another ticket for cleaning up all the license checks for pro and enterprise:  https://mattermost.atlassian.net/browse/MM-58700

## Ticket Link
https://mattermost.atlassian.net/browse/MM-58377

### CODE REVIEW

File: server/api_playbooks_test.go [hunk 1]
```diff
@@ -98,6 +65,7 @@ func TestPlaybooks(t *testing.T) {
 			Title:  "test6 - to be archived",
 			TeamID: e.BasicTeam.Id,
 		})
+		t.Log("err is", err)
```
Comment (mgdelacroix) [inline server/api_playbooks_test.go:68]: nit: we can probably remove this
```suggestion
```

**Review by mgdelacroix — APPROVED** [review]

File: server/api_playbooks_test.go (hunk 1 above)
Comment (JulienTant) [reply server/api_playbooks_test.go]: Ooopsie! Thanks :D

**Review by calebroseland — APPROVED** [review]

**Pull request merged**